web/viewer.html

```diff
@@ -263,18 +289,6 @@
                 <span id="numPages" class="toolbarLabel"></span>
               </div>
               <div id="toolbarViewerRight">
-                <div id="editorModeButtons" class="splitToolbarButton toggled hidden" role="radiogroup">
-                  <button id="editorNone" class="toolbarButton toggled" title="Disable Annotation Editing" role="radio" aria-checked="true" tabindex="31" data-l10n-id="editor_none">
-                    <span data-l10n-id="editor_none_label">Disable Editing</span>
-                  </button>
-                  <button id="editorFreeText" class="toolbarButton" title="Add FreeText Annotation" role="radio" aria-checked="false" tabindex="32" data-l10n-id="editor_free_text">
-                    <span data-l10n-id="editor_free_text_label">FreeText Annotation</span>
-                  </button>
-                  <button id="editorInk" class="toolbarButton" title="Add Ink Annotation" role="radio" aria-checked="false" tabindex="33" data-l10n-id="editor_ink">
-                    <span data-l10n-id="editor_ink_label">Ink Annotation</span>
-                  </button>
-                </div>
-
                 <button id="presentationMode" class="toolbarButton hiddenLargeView" title="Switch to Presentation Mode" tabindex="43" data-l10n-id="presentation_mode">
```

Comment:
Unfortunately all of the `tabindex`s in the `toolbarViewerRight`-div now needs to be updated again, really sorry about that!
Hence this button should have `tabindex="31"`, with the following buttons numbered sequentially.

---

Previously, when I asked you to update the `tabindex`s, I didn't know that you intended to add new "toolbars" from the editing buttons. Had that been clear to me, I'd definitely have recommended placing the editing buttons in their new position (just before the SecondaryToolbarToggle) from the start.